spezi-invitation-code/index.js

```diff
@@ -0,0 +1,72 @@
+//
+// This source file is part of the StudyApplication based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+const admin = require("firebase-admin");
+const {https} = require("firebase-functions/v2");
+const {FieldValue} = require("firebase-admin/firestore");
+
+class InvitationCodeVerifier {
+  constructor(firestore) {
```

Comment:
One thing I could see useful is a regex for the invitation code that is checked in the first function? Can be nil by default and not checked if one is not configured but could be a useful extension: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_expressions